tests/test_hedera.py

```diff
@@ -35,6 +35,9 @@ def test_hedera_get_public_key_ok(backend, firmware, navigator, test_name):
             elif backend.firmware.device.startswith("nano"):
                 nav_ins = [NavInsID.RIGHT_CLICK,
                            NavInsID.BOTH_CLICK]
+            else:
+                nav_ins = [NavInsID.USE_CASE_CHOICE_CONFIRM,
+                           NavInsID.USE_CASE_ADDRESS_CONFIRMATION_CONFIRM]
```

Comment:
Not really necessary as you are using "function" scope, but I think it would be better to end all stax navigation instructions list with `NavInsID.USE_CASE_STATUS_DISMISS` so that we are still back on home screen.

Reply:
Done